src/main/java/org/sciborgs1155/robot/intake/Intake.java

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.sciborgs1155.robot.intake;
+
+import static org.sciborgs1155.robot.Ports.Intake.*;
+import static org.sciborgs1155.robot.intake.IntakeConstants.*;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.State;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.TrapezoidProfileCommand;
+import org.sciborgs1155.lib.constants.SparkUtils;
+
+public class Intake extends SubsystemBase {
+
+  private final CANSparkMax rotateMotor =
+      SparkUtils.create(
+          ROTATION_MOTOR,
+          s -> {
+            s.setIdleMode(IdleMode.kBrake);
+            s.setSmartCurrentLimit(40);
+            s.setOpenLoopRampRate(0);
+          });
+
+  private final CANSparkMax wheels =
+      SparkUtils.create(
+          WHEEL_MOTOR,
+          s -> {
+            s.setIdleMode(IdleMode.kBrake);
+            s.setSmartCurrentLimit(40);
+            s.setOpenLoopRampRate(0);
+          });
+
+  private final PIDController pid;
+  private final ArmFeedforward ff;
+  private final Encoder rotationEncoder;
+
+  public Intake() {
+    rotationEncoder = new Encoder(ENCODER[0], ENCODER[1]);
```

Comment:
The WPILib `Encoder` class refers to a quadrature encoder plugged into the rio. I believe (really hope) we are using a 1:1 geared throughbore encoder for any arm-like mechanism (such as the intake), since it can be used in code with the `DutyCycleEncoder` class.